app/views/board/bits.scala

```diff
@@ -127,23 +127,33 @@ object bits {
 
   def jsData(
       sit: Situation,
-      fen: FEN
+      fen: FEN,
+      variant: Variant = Variant.libStandard(GameLogic.Chess())
   )(implicit ctx: Context) =
     Json.obj(
       "fen"         -> fen.value, // require full fen for score updates
-      "baseUrl"     -> s"$netBaseUrl${routes.Editor.load("")}",
+      "baseUrl"     -> s"${netBaseUrl}/editor/",
       "playerIndex" -> sit.player.letter.toString,
       "castles" -> Json.obj(
         "K" -> sitCanCastle(sit, P1, strategygames.chess.KingSide),
         "Q" -> sitCanCastle(sit, P1, strategygames.chess.QueenSide),
         "k" -> sitCanCastle(sit, P2, strategygames.chess.KingSide),
         "q" -> sitCanCastle(sit, P2, strategygames.chess.QueenSide)
       ),
-      "animation" -> Json.obj("duration" -> ctx.pref.animationMillis),
-      "is3d"      -> ctx.pref.is3d,
-      "i18n"      -> i18nJsObject(i18nKeyes)
+      "animation"  -> Json.obj("duration" -> ctx.pref.animationMillis),
+      "is3d"       -> ctx.pref.is3d,
+      "i18n"       -> i18nJsObject(i18nKeyes),
+      "variantKey" -> formatVariantKey(variant.key)
     )
 
+  private def formatVariantKey(key: String): String =
```

Comment:
we ought to abstract this into strategy games as its used in a few places. Not sure if we have one in the front end going from lower case to respective keys.